### PR TITLE
[CI] [GHA] Use Dockerfiles for fedora and webassembly workflows

### PR DESCRIPTION
**Pull request by akashchi**
### Tickets:
 - *145562*

### CODE REVIEW

File: .github/workflows/fedora.yml [hunk 1]
```diff
@@ -38,17 +39,44 @@ jobs:
           component_pattern: "category: (.*)"
           repo_token: ${{ secrets.GITHUB_TOKEN }}
           skip_when_only_listed_labels_set: 'docs'
-          skip_when_only_listed_files_changed: '*.md,*.rst,*.png,*.jpg,*.svg,*/layer_tests_summary/*,*/conformance/*'
```
Comment (akladiev) [inline .github/workflows/fedora.yml:41]: By mistake?

Comment (akashchi) [reply .github/workflows/fedora.yml:41]: Yes, thanks, fixed for both workflows.

**Review by akladiev — APPROVED** [review]

File: .github/workflows/fedora.yml [hunk 2]
```diff
@@ -92,11 +117,6 @@ jobs:
       - name: Install build dependencies
         run: bash ${OPENVINO_REPO}/install_build_dependencies.sh
```
Comment (mryzhov) [inline .github/workflows/fedora.yml:118]: why we do not install them in the docker images?

**Review by mryzhov — APPROVED** [review]

File: .github/workflows/fedora.yml (hunk 2 above)
Comment (akashchi) [reply .github/workflows/fedora.yml]: They are installed there, forgot to remove it from the workflow file; removed.

**Pull request merged**